.github/workflows/build_wheels_linux_arm.yml

```diff
@@ -0,0 +1,167 @@
+name: Build PYPI wheels for opencv-python on Linux ARM
+
+on:
+  pull_request:
+    branches:
+      - master
+  release:
+    types: [published, edited]
+
+
+jobs:
+  build_arm:
+    runs-on: ${{ matrix.os }}
+    defaults:
+      run:
+        shell: bash
+
+    strategy:
+      fail-fast: false
```

Comment:
why not fail fast?

Reply:
I wanted to keep it for waiting other builds, to be sure that the fail happened only on a specific build, but not on all builds. In this case, we can keep running a pipeline with several builds, but when only one failed with a network issue or with something else.